packages/databricks-vscode/src/bundle/GenerateBundle.ts

```diff
@@ -1,16 +1,64 @@
 import {CliWrapper} from "../cli/CliWrapper";
 import {extensions, Uri} from "vscode";
-import path from "node:path";
+import {workspace, RelativePattern, GlobPattern, Disposable} from "vscode";
+import YAML from "yaml";
 
-export async function generateBundleSchema(cli: CliWrapper) {
+export async function generateBundleSchema(
+    cli: CliWrapper
+): Promise<Disposable> {
     // get freshly generated bundle schema
     const bundleSchema = await cli.getBundleSchema();
 
     // URI scheme for DABs JSON schemas
     const dabsUriScheme = "dabs";
 
     // URI for bundle root config json schema
-    const rootConfigSchemaUri = `${dabsUriScheme}:///root.json`;
+    const rootConfigSchemaUri = `${dabsUriScheme}:///dabs.json`;
+
+    const folder = workspace.workspaceFolders?.[0];
+    const configFilePattern = new RelativePattern(
+        folder!,
```

Comment:
Return if no folder. Failing gracefully is better.

Reply:
Aren't we guaranteed to always have a workspace folder? 

https://github.com/databricks/databricks-vscode/blob/main/packages/databricks-vscode/src/extension.ts#L71-L84

Still, I agree that we should make this code more defensive.

Reply:
Yep. Just do not like the look of `!`

Reply:
Agreed. I'll change it